tasks/go.py

```diff
@@ -102,7 +102,7 @@ def vet(ctx, targets):
     # add the /... suffix to the targets
     args = ["{}/...".format(t) for t in targets]
     build_tags = get_default_build_tags()
-    ctx.run("go vet -x -tags \"{}\" ".format(build_tags) + " ".join(args))
+    ctx.run("go vet -tags \"{}\" ".format(build_tags) + " ".join(args))
```

Comment:
what did the x do?

Reply:
it prints out the path of every file it examines

Reply:
Oh! yea this is good :) thanks!